taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from taxi.views import index
+from django.urls import path
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports.